homeassistant/components/dnsip/config_flow.py

```diff
@@ -0,0 +1,78 @@
+"""Adds config flow for dnsip integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_HOSTNAME,
+    CONF_IPV6,
+    CONF_RESOLVER,
+    CONF_RESOLVER_IPV6,
+    DEFAULT_HOSTNAME,
+    DEFAULT_IPV6,
+    DEFAULT_NAME,
+    DEFAULT_RESOLVER,
+    DEFAULT_RESOLVER_IPV6,
+    DOMAIN,
+)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOSTNAME, default=DEFAULT_HOSTNAME): cv.string,
+        vol.Optional(CONF_RESOLVER, default=DEFAULT_RESOLVER): cv.string,
+        vol.Optional(CONF_IPV6, default=DEFAULT_IPV6): cv.boolean,
+        vol.Optional(CONF_RESOLVER_IPV6, default=DEFAULT_RESOLVER_IPV6): cv.string,
+    }
+)
+
+
+class DnsIPConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for dnsip integration."""
+
+    VERSION = 1
+
+    async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
+        """Import a configuration from config.yaml."""
+
+        self.context.update({"title_placeholders": {"DNS IP": f"YAML import {DOMAIN}"}})
```

Comment:
This context isn't needed on import? As in, we won't show it to the user anyways?